src/commands/general/botstats.js

```diff
@@ -0,0 +1,42 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { MessageEmbed } from 'discord.js';
+
+export const data = new SlashCommandBuilder()
+	.setName('botstats')
+	.setDescription('Information about the current statistics of the bot');
+
+export async function execute(interaction, client) {
+	try {
+		let uptime = await client.tools.convertTime(client.uptime);
+		let ram = (
+			process.memoryUsage().heapUsed / 1024 / 1024 +
+			process.memoryUsage().heapTotal / 1024 / 1024
```

Comment:
You're calling a possibly heavy call twice. Might be better to call it once and store in a const!